### ISSUE TITLE
pool key needs to be 48bit

### ISSUE THREAD

**Issue by acazmaj144** [madMAx43v3r/chia-plotter#132]
i have problem i installed everything right and it shows me this

aca@zmaj:~/chia-plotter$ ./build/chia_plot -p 87cdb980d7a17fb87ccfeb2948f2051471cad075efbd0657a1aa5198b9fe3757efde -f b7f6ee75bf85b9a0b92072632d058dd1984e7784d5ddc7868b8d793d0e84ceca11 -t /media/aca/aca2tb/ -2 /media/aca/New Volume/ -d /media/aca/S1 K F/ -n 1 -r 6
Invalid <poolkey>: 87cdb980d7a17fb87ccfeb2948f2051471cad075efbd0657a1aa5198b9fe3757efde, '87cdb980d7a17fb87ccfeb2948f2051471cad075efbd0657a1aa5198b9fe3757efde' (needs to be 48 bytes)

i checked everything and everything is right any help?

**Comment by xiaoyin199** [comment]
I have the same problem

**Comment by xiaoyin199** [comment]
help！

**Comment by madMAx43v3r** [comment]
```
echo 87cdb980d7a17fb87ccfeb2948f2051471cad075efbd0657a1aa5198b9fe3757efde | wc -c
69
```
48 bytes => should be 96 characters per key, your keys are too short

**Comment by madMAx43v3r** [comment]
check `chia keys show` for the keys

**Comment by acazmaj144** [comment]
aca@zmaj:~/chia-plotter$ echo 87cdb980d7a17fb87ccfeb2948f2051471cad075efbd0657a1aa5198b9fe3757efde | wc -c
69
aca@zmaj:~/chia-plotter$ 69
69: command not found
aca@zmaj:~/chia-plotter$ chia keys show

Command 'chia' not found, did you mean:

  command 'chi2' from deb paml (4.9j+dfsg-1)

Try: sudo apt install <deb name>

aca@zmaj:~/chia-plotter$

**Comment by acazmaj144** [comment]
when i try chia keys show its giving me something wrong

**Comment by VertiHydro** [comment]
You actually have to install the chia blockchain somewhere first - do you have a Chia wallet? If so, on what computer did you create your wallet?

**Comment by acazmaj144** [comment]
yes i have installed chia blockchain and plotter 12 plots i have wallet i created wallet on other windows pc

**Comment by acazmaj144** [comment]
1.1.5

**Comment by acazmaj144** [comment]
i made a new wallet here on my linux pc and same again

**Comment by madMAx43v3r** [comment]
@acazmaj144 if those are your private keys, you'll have to make a new wallet now, and transfer any XCH to the new one

**Comment by acazmaj144** [comment]
i have no xch anywhere

**Comment by madMAx43v3r** [comment]
I removed the keys, still somebody could have copied them, and any XCH you receive in the future will be lost

**Comment by VertiHydro** [comment]
ok so yep you maybe just exposed your wallet so you're going to need to make a new one - first update Chia Blockchain from their website https://www.chia.net/

**Comment by acazmaj144** [comment]
it shows on my linux pc chia blockchain version 0.0

**Comment by gmorb** [comment]
Having same issue chia keys show is not working for me i even installed the chia blockchain.

**Comment by acazmaj144** [comment]
0 info it worked if it ends with ok
1 verbose cli [ '/usr/bin/node', '/usr/bin/npm', 'run', 'electron' ]
2 info using npm@6.14.4
3 info using node@v10.19.0
4 verbose config Skipping project config: /home/aca/.npmrc. (matches userconfig)
5 verbose stack Error: ENOENT: no such file or directory, open '/home/aca/package.json'
6 verbose cwd /home/aca
7 verbose Linux 5.8.0-55-generic
8 verbose argv "/usr/bin/node" "/usr/bin/npm" "run" "electron"
9 verbose node v10.19.0
10 verbose npm  v6.14.4
11 error code ENOENT
12 error syscall open
13 error path /home/aca/package.json
14 error errno -2
15 error enoent ENOENT: no such file or directory, open '/home/aca/package.json'
16 error enoent This is related to npm not being able to find a file.
17 verbose exit [ -2, true ]

**Comment by acazmaj144** [comment]
issue when trying to install chia blockchain

**Comment by gmorb** [comment]
> issue when trying to install chia blockchain

You doing purely command line? Theres a gui version as well. Keeping posted to see what solution you get and try and apply it to my issue. 😅

**Comment by danielvm-git** [comment]
If you used the .deb to install the GUI on linux your installation is at /usr/lib/chia-blockchain/resources/app.asar.unpacked/daemon/

try like this:

cd /usr/lib/chia-blockchain/resources/app.asar.unpacked/daemon/
./chia keys show

it will show your keys and you should be fine

**Comment by szarkivach** [comment]
> I had same problem. Found the problem! If your temp drive or destination drive label name has a space you get that error. Rename drives to one word no spaces and boom!
